ui/src/state/diary/diary.ts

```diff
@@ -160,6 +170,25 @@ export function useNotes(flag: DiaryFlag) {
   };
 }
 
+export function useNotesOnHost(flag: DiaryFlag): DiaryOutlines | undefined {
+  const notes = useRef<DiaryOutlines | undefined>(undefined);
+
+  useEffect(() => {
+    const getNotes = async () =>
+      api.scry({
+        app: 'diary',
+        path: `/diary/${flag}/notes/newest/${INITIAL_MESSAGE_FETCH_PAGE_SIZE}/outline`,
+      });
+    getNotes().then((n) => {
+      if (n !== null && typeof n === 'object') {
+        notes.current = n as DiaryOutlines;
+      }
+    });
+  }, [flag]);
+
+  return notes.current;
+}
```

Comment:
This should probably use react query with a special key for host only. as is if this takes too long notes will just be undefined by the time it gets used, and since this uses a ref it won't trigger re-renders to recheck w/e is using this.